test/models/test_nn.py

```diff
@@ -64,23 +77,45 @@ def test_DenseSymm(symmetries, use_bias):
 @pytest.mark.parametrize("symmetries", ["trans", "autom"])
 @pytest.mark.parametrize("use_bias", [True, False])
 @pytest.mark.parametrize("lattice", [nk.graph.Chain, nk.graph.Square])
-def test_DenseEquivariant(symmetries, use_bias, lattice):
+@pytest.mark.parametrize("mode", ["fft", "matrix", "irrep"])
+def test_DenseEquivariant(symmetries, use_bias, lattice, mode):
     rng = nk.jax.PRNGSeq(0)
 
     g, hi, perms = _setup_symm(symmetries, N=3, lattice=lattice)
 
     pt = perms.product_table
     n_symm = np.asarray(perms).shape[0]
 
-    ma = nk.nn.DenseEquivariant(
-        symmetry_info=pt.ravel(),
-        in_features=1,
-        out_features=1,
-        use_bias=use_bias,
-        bias_init=nk.nn.initializers.uniform(),
-    )
+    if mode == "irrep":
+        ma = nk.nn.DenseEquivariant(
+            symmetries=perms,
+            mode=mode,
+            in_features=1,
+            out_features=1,
+            use_bias=use_bias,
+            bias_init=nk.nn.initializers.uniform(),
+        )
+    elif mode == "fft":
+        ma = nk.nn.DenseEquivariant(
+            symmetries=pt,
+            shape=tuple(g.extent),
+            mode=mode,
+            in_features=1,
+            out_features=1,
+            use_bias=use_bias,
+            bias_init=nk.nn.initializers.uniform(),
+        )
+    else:
+        ma = nk.nn.DenseEquivariant(
+            symmetries=pt,
+            mode=mode,
+            in_features=1,
+            out_features=1,
+            use_bias=use_bias,
+            bias_init=nk.nn.initializers.uniform(),
+        )
```

Comment:
```suggestion
```

Reply:
This is the test for mode="matrix", the original way I implemented the DenseEquivariant layers. 

The modes "irreps" and "fft" are much faster, but I wanted to keep this implementation around in case there is an unanticipated use case down the line. (i.e. there's a group where there is no translational symmetries and no way to generate irreps).

Reply:
Ahh I see issue, fixing it

Reply:
it's simply more concise. you don;t need to use different constructors to construct them.
The same line above can be used to construct both.

Reply:
Wait actually, maybe I just need to add a kwargs pointer for the flax modules

Reply:
why not? whats the error?

Reply:
I just tried to replace this with

```
def test_DenseEquivariant(symmetries, use_bias, lattice, mode):
    rng = nk.jax.PRNGSeq(0)

    g, hi, perms = _setup_symm(symmetries, N=3, lattice=lattice)

    pt = perms.product_table
    n_symm = np.asarray(perms).shape[0]

    if mode == "irreps":
        ma = nk.nn.DenseEquivariant(
            symmetries=perms,
            mode=mode,
            in_features=1,
            out_features=1,
            use_bias=use_bias,
            bias_init=nk.nn.initializers.uniform(),
        )
    else:
        ma = nk.nn.DenseEquivariant(
            symmetries=pt,
            shape=tuple(g.extent),
            mode=mode,
            in_features=1,
            out_features=1,
            use_bias=use_bias,
            bias_init=nk.nn.initializers.uniform(),
        )

    pars = ma.init(rng.next(), np.random.normal(0, 1, [1, 1, n_symm]))
```

and it works. 
Again i don't see the errors

Reply:
I should clarify that I don't know how to do it. Just placing **kwargs after the argument returns a syntax error